dev-tools/reconfigurator-cli/tests/output/cmd-set-mgs-updates-stdout

```diff
@@ -0,0 +1,2835 @@
+
+
+    omicron zones:
+    ---------------------------------------------------------------------------------------------------------------
+    zone type         zone id                                image source      disposition   underlay IP           
+    ---------------------------------------------------------------------------------------------------------------
+    crucible          327db2a9-fc45-4c3f-a6db-4ee58459e5cf   install dataset   in service    fd00:1122:3344:103::27
+    crucible          5ea7f627-d51e-4e2d-8648-112ddc635e53   install dataset   in service    fd00:1122:3344:103::25
+    crucible          ad81aafc-e74d-4b0a-b4e6-2e58ef52b7a2   install dataset   in service    fd00:1122:3344:103::26
+    crucible_pantry   a76d9883-b4fe-464f-acbb-e75b2bbb508d   install dataset   in service    fd00:1122:3344:103::24
+    external_dns      8a651785-3936-4cf6-b301-4fa7fa003c9f   install dataset   in service    fd00:1122:3344:103::23
+    internal_dns      f14b0fff-c0f0-420f-ab51-df1a605f43c3   install dataset   in service    fd00:1122:3344:3::1   
+    internal_ntp      f282c429-5f5e-4430-8f71-600cf61ded56   install dataset   in service    fd00:1122:3344:103::21
+    nexus             1dee8f37-8286-48dc-b4b4-e59979c030ec   install dataset   in service    fd00:1122:3344:103::22
+
+
+ COCKROACHDB SETTINGS:
+    state fingerprint:::::::::::::::::   (none)
+    cluster.preserve_downgrade_option:   (do not modify)
+
+ OXIMETER SETTINGS:
+    generation:   1
+    read from::   SingleNode
+
+ METADATA:
+    created by:::::::::::   reconfigurator-cli
+    created at:::::::::::   <REDACTED_TIMESTAMP>
+    comment::::::::::::::   (none)
+    internal DNS version:   1
+    external DNS version:   1
+
+ PENDING MGS-MANAGED UPDATES: 1
+    Pending MGS-managed updates (all baseboards):
+    ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+    sp_type   slot   part_number   serial_number   artifact_kind   artifact_hash                                                      artifact_version   details                                                                                                               
+    ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+    sled      2      model2        serial2         gimlet_sp       e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855   1.1.0              Sp { expected_active_version: ArtifactVersion("1.0.0"), expected_inactive_version: Version(ArtifactVersion("1.0.1")) }
```

Comment:
The details here are very long. If not too onerous, I'd consider splitting them up into an `expected active vsn` and `expected inactive vsn` columns. 

This only makes sense though if the details are going to always look like this, which presumably they might not given the naming.

Reply:
Right, it's not always going to look like this.  Even though the RoT also has two slots and one is active, we're going to want to be specific about what's in slot A and slot B and which is active, rather than saying "this is what the active slot should be [and I don't care if that's the A slot or B slot".  For the RoT bootloader, there might be different data here.

I agree this is unwieldy and it might be useful to have a multi-line row format but that seems out of scope here.

Reply:
Makes sense. Let's keep it as is for now.